pyproject.toml

```diff
@@ -27,7 +27,19 @@ classifiers = [
 ]
 dynamic = ["version"]
 dependencies = [
-  "bioio-base>=0.2.0",
+  "bioio-base>=0.3.0",
+  "dask[array]>=2021.4.1",
+  "fsspec>=2022.8.0",
+  "imagecodecs>=2020.5.30",
+  "numpy>=1.16",
```

Comment:
`1.21.0` is the minimum version supporting our minimum python version (3.9) I believe

Reply:
resolved in [d07550a](https://github.com/bioio-devs/bioio-ome-tiff/pull/2/commits/d07550a7c2d5aac2de6a00a54ade1fc60281badd)